BasicSamples/ButtonClicker/build.gradle

```diff
@@ -11,6 +11,11 @@ android {
         versionCode 1
         versionName "1.0"
     }
+
+    lintOptions {
+        abortOnError false
```

Comment:
Should this be true (is that the default?) and fix the errors, or if false is the default it does not need to be here?